bigbone/src/main/kotlin/social/bigbone/api/entity/Instance.kt

```diff
@@ -284,5 +312,156 @@ data class Instance(
+    )
+
+    /**
+     * Hints related to contacting a representative of the website.
+     */
+    @Serializable
+    data class Contact(
+        /**
+         * An email address that can be messaged regarding inquiries or issues.
+         */
+        @SerialName("email")
+        val email: String = "",
+
+        /**
+         * An account that can be contacted natively over the network regarding inquiries or issues.
+         */
+        @SerialName("account")
+        val account: Account = Account()
+    )
+}
+
+/**
+ * Instance activity class that can be returned by e.g. <a href="https://docs.joinmastodon.org/methods/instance/#activity">methods/instance/#activity</a>
+ */
+@Serializable
+data class InstanceActivity(
+    /**
+     * String (UNIX Timestamp). Midnight at the first day of the week.
+     */
+    @SerialName("week")
+    val week: String = "",
+    /**
+     * String (cast from an integer). The number of Statuses created since the week began.
+     */
+    @SerialName("statuses")
+    val statuses: String = "",
+    /**
+     * String (cast from an integer). The number of user logins since the week began.
+     */
+    @SerialName("logins")
+    val logins: String = "",
+    /**
+     * String (cast from an integer). The number of user registrations since the week began.
+     */
+    @SerialName("registrations")
+    val registrations: String = ""
+)
+
+/**
+ * Represents a domain that is blocked by the instance.
+ * @see <a href="https://docs.joinmastodon.org/entities/DomainBlock/">entities/DomainBlock</a>
+ */
+@Serializable
+data class DomainBlock(
+    /**
+     * The domain which is blocked. This may be obfuscated or partially censored.
+     */
+    @SerialName("domain")
+    val domain: String = "",
+
+    /**
+     * The SHA256 hash digest of the domain string.
+     */
+    @SerialName("digest")
+    val digest: String = "",
+
+    /**
+     * The level to which the domain is blocked.
+     */
+    @SerialName("severity")
+    val severity: Severity = Severity.SILENCE,
+
+    /**
+     * An optional reason for the domain block.
+     */
+    @SerialName("comment")
+    val comment: String? = null
+) {
+    /**
+     * The level to which the domain is blocked.
+     */
+    @Serializable
+    enum class Severity {
+        /**
+         * Users from this domain will be hidden from timelines, threads, and notifications (unless you follow the user).
+         */
+        @SerialName("silence")
+        SILENCE,
+
+        /**
+         * Incoming messages from this domain will be rejected and dropped entirely.
+         */
+        @SerialName("suspend")
+        SUSPEND
     }
 }
+
+/**
+ * Represents an extended description for the instance, to be shown on its about page.
+ */
+@Serializable
+data class ExtendedDescription(
+    /**
+     * A timestamp of when the extended description was last updated.
+     * String (ISO 8601 Datetime)
+     */
+    @SerialName("updated_at")
+    val updatedAt: String = "",
```

Comment:
Thinking about these ISO8601 formatted date strings we send and receive; Wouldn't it be nice, if we could directly convert this into an `Instant` instead? This would relieve BigBone users from having to do it and make using our lib more convenient. 

If we plan to to this, this should happen as part of a separate PR and the whole set of entities.

Reply:
Yep, I was thinking the same when I saw these date strings.

I’ll create a ticket and get on with implementing that next! 😊